front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -62,9 +64,10 @@ function CreateShipmentView() {
   const { triggerError } = useErrorHandling();
   const { createToast } = useNotification();
   const { globalPreferences } = useContext(GlobalPreferencesContext);
```

Comment:
as mentioned https://github.com/boxwise/boxtribute/pull/1496#discussion_r1740609085

we could remove the usage of globalPreferences all together so that the loading state is not even necessary. Would do  so now.

Reply:
The organization id is used throughout this view, so we still need to keep it there. However, as I mentioned in the comment following yours in https://github.com/boxwise/boxtribute/pull/1496#discussion_r1742971510, there's a task to address the global context refactor.

Reply:
ok, then let's leave it for later. thought it is just in one occasion